boards/arm/mps2_an521/doc/index.rst

```diff
@@ -0,0 +1,448 @@
+.. _mps2_an521_board:
+
+ARM MPS2+ AN521
+###############
+
+Overview
+********
+
+The mps2_an521 board configuration is used by Zephyr applications that run
+on the MPS2+ AN521 board. It provides support for the MPS2+ AN521 ARM Cortex-M33
+CPU and the following devices:
+
+- Nested Vectored Interrupt Controller (NVIC)
+- System Tick System Clock (SYSTICK)
+- Cortex-M System Design Kit GPIO
+- Cortex-M System Design Kit UART
+
+.. image:: img/mps2_an521.png
+     :width: 666px
+     :align: center
+     :height: 546px
+     :alt: ARM MPS2+ AN521
+
+More information about the board can be found at the `MPS2 FPGA Website`_.
+
+Hardware
+********
+
+ARM MPS2+ AN521 provides the following hardware components:
+
+
+
+- ARM Cortex-M33
+- Soft Macro Model (SMM) implementation of SSE-200 subsystem
+- Memory
+
+  - 16MB internal memory SRAM.
```

Comment:
LGTM.   My only issue would be that list items (such as these) that aren't sentences, shouldn't end with a period

Reply:
Updated, remove all the "." for the list items.
Thanks.